operator/src/lib.rs

```diff
@@ -57,38 +55,30 @@ pub enum KafkaNodeType {
 struct KafkaState {
     context: ReconciliationContext<KafkaCluster>,
     kafka_cluster: KafkaCluster,
-    zk_cluster: Option<ZooKeeperCluster>,
+    zoo_keeper_info: Option<stackable_zookeeper_crd::util::ZookeeperConnectionInformation>,
```

Comment:
Ok i adapt it monday :)

Reply:
Im not sure about that.
in crd/src/lib.rs he refers to zoo_keeper_reference. I wanted to keep it consistent. @soenkeliebau  what do you prefer?

Reply:
I'd argue that they are not my naming rules, I'm merely trying to apply _our_ naming rules in a consistent way :)

I am happy either way, but if we choose zoo_keeper_... then we could also argue about `h_base_...` or `ni_fi_...`

To me personally it makes sense to treat the product name as a single token for camel- or snake-case. 

The var `zoo_keeper_reference` in _crd/src/lib.rs_ was introduced by Lars and correct at the time, because the struct was still named `ZooKeeperCluster` - if we change that this should in time also be renamed.

Reply:
Ok i prefer to keep the names together as well, then ill adapt them to zookeeper_xyz.